src/zzz_od/gui/view/home/home_interface.py

```
@@ -370,16 +401,41 @@ def _extract_image_url(self, data):
+                            return video_url
         return None
 
     def _save_image(self, content):
-        """保存图片"""
-        temp_path = self.save_path + '.tmp'
-        with open(temp_path, "wb") as f:
-            f.write(content)
-        if os.path.exists(self.save_path):
-            os.remove(self.save_path)
-        os.rename(temp_path, self.save_path)
+        """保存图片/视频，确保临时文件被正确清理"""
```

Ruff (0.14.0)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)